tests/test_generator.py

```diff
@@ -43,3 +44,12 @@ def test_identify(self):
         assert parse_one("X").sql(identify="safe") == "X"
         assert parse_one("x as 1").sql(identify="safe") == '"x" AS "1"'
         assert parse_one("X as 1").sql(identify="safe") == 'X AS "1"'
+
+    def test_pad_comment(self):
+        self.assertEqual(Generator().pad_comment(""), "")
+        self.assertEqual(Generator().pad_comment(" leading"), " leading ")
+        self.assertEqual(Generator().pad_comment("trailing "), " trailing ")
+        self.assertEqual(
+            Generator().pad_comment(" leading and trailing "), " leading and trailing "
+        )
+        self.assertEqual(Generator().pad_comment("no padding"), " no padding ")
```

Comment:
Can we instead test this end-to-end by simply adding an example that fails to roundtrip in pretty.sql?